app/src/main/java/com/example/hungrywolfs/SplashScreenActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.hungrywolfs
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+
+private const val SPLASH_TIME: Long = 1500
+
+class SplashScreenActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        Handler().postDelayed({
```

Comment:
Instead of Handlers, which are the old way of doing Async stuff, you can use coroutines here, specifically **lifecycleScope** which is built in into Fragments and add a delay before opening a new activity

https://developer.android.com/topic/libraries/architecture/coroutines#lifecyclescope